lms/djangoapps/certificates/views.py

```diff
@@ -224,3 +232,227 @@ def update_example_certificate(request):
 
     # Let the XQueue know that we handled the response
     return JsonResponse({'return_code': 0})
+
+
+# pylint: disable=too-many-statements, bad-continuation
+@login_required
+def render_html_view(request):
+    """
+    This view generates an HTML representation of the specified student's certificate
+    If a certificate is not available, we display a "Sorry!" screen instead
+    """
```

Comment:
One note about the JSON failover data -- we could certainly drop this in favor of a data migration, although i'm still not clear on how to control the feature using the configuration's enabled/disabled flag.  It does sound really nice to be able to control the availability of features at runtime w/o having to modify config files, restart services, all that.